benchmarks/s3.properties

```diff
@@ -0,0 +1,77 @@
+# ============================================= BlobStore Implementation ==================================
```

Comment:
May I ask where this file is used?

(I did not understood)

Reply:
it may be called in the groovy

Reply:
`-v benchmarks/${params.PROFILE}.properties:/root/conf/blobs.properties`

By the monts, got it.